src/System.Linq.Expressions/src/System/Linq/Expressions/MemberExpression.cs

```diff
@@ -179,6 +179,7 @@ public static MemberExpression Field(Expression expression, FieldInfo field)
         public static MemberExpression Field(Expression expression, string fieldName)
         {
             RequiresCanRead(expression, nameof(expression));
+            ContractUtils.RequiresNotNull(fieldName, nameof(fieldName));
```

Comment:
This changes the parameter name only. Without it you get the same exception but an empty Parm name  - I put it in here to be consistent with Property that checks before calling Type.GetProperty